packages/static-module-record/test/test-static-module-record.js

```diff
@@ -792,3 +792,12 @@ test('should handle package "immer" source', t => {
     setUseProxies: ['vn', true],
   });
 });
+
+// https://github.com/endojs/endo/issues/2094
```

Comment:
Weird!

Is this a drive-by, or is this new test in this PR for a deeper reason?

Reply:
I’ve bundled this test in here to explain why I did not use this pattern in the fix. My first attempt was:

```js
const {
  // ...
  quote,
  bare = quote,
  // ...
} = globalThis.assert;

export {
  // ...
  quote,
  bare,
  quote as q,
  bare as b,
  // ...
};
```

Reply:
Notably, I detected this bug because a pet daemon test tripped on it. It manifests as a deeply nested Babel “Can’t get property buildError of undefined” exception.

Reply:
Ok. Resolving.